docs/articles/documentation/guides/advanced-guides/test-https-features-and-http2-websites.md

```diff
@@ -11,6 +11,8 @@ redirect_from:
 This topic describes how TestCafe works with HTTPS and HTTP/2 websites.
```

Comment:
"how TestCafe works with" What do you mean when you say 'works with'?

Reply:
TestCafe hosts a proxy and interacts with the tested website from there. When i say 'works with' I mean the whole process- from enabling the proxy to use HTTPS, to running actual test scripts

Reply:
It is better to be more specific instead of saying 'how it works with' something